go/src/socialapi/workers/payment/tests.sh

```diff
@@ -3,11 +3,6 @@
 echo "running payment tests"
 go test -c socialapi/workers/payment/ && ./payment.test -c $(dirname $0)/../../config/dev.toml -test.v=true
 
-echo "running stripe tests"
-go test -c socialapi/workers/payment/stripe/ && ./stripe.test -c $(dirname $0)/../../config/dev.toml -test.v=true
+echo "running api tests"
+go test -c socialapi/workers/payment/api && ./api.test -c $(dirname $0)/../../config/dev.toml -test.v=true
```

Comment:
More elegant imo is to make test read flags params after `--`:

```
go test -v -race socialapi/workers/payment/api -- -c $(dirname $0)/../../config/dev.toml
```

in tests:

```go
var testArgs []strong

func init() {
    for i, arg := range os.Args {
        if arg == "--" {
            testArgs = os.Args[i+1:]
            break
        }
    }
}

...
flag.CommandLine.Parse(testArgs)
...
```

Just an idea.